pkg/scheduler/scheduler.go

```diff
@@ -476,13 +478,14 @@ func (s *Scheduler) scheduleResourceBinding(resourceBinding *workv1alpha2.Resour
 	}
 
 	scheduleResult, err := s.Algorithm.Schedule(context.TODO(), &placement, &resourceBinding.Spec, &core.ScheduleAlgorithmOption{EnableEmptyWorkloadPropagation: s.enableEmptyWorkloadPropagation})
-	if err != nil {
+	if err != nil && reflect.TypeOf(err) != reflect.TypeOf(&framework.FitError{}) {
```

Comment:
I'm afraid we have a situation here. When a cluster is not ready for a while, the cluster will be added a NoSchedule taint. And before, a workload is running on this cluster and happens to scale up(may be triggered by users or hpa-controller), however, no cluster is fit due to the taint. I guess the previous replicas will be evicted here because now fit error is skipped and result will be patched to RB.

Reply:
Oh, I see. I didn't realize taint plugin is updated here. Thanks.

Reply:
Now the `FitError` will be dedicated used in the situation that `no cluster fit`, right?

Reply:
yes, then the `fitError` information will be shown in the event and RB's status, the same behavior as k8s.